src/target/riscv/riscv-013.c

```diff
@@ -1272,193 +1286,224 @@ static bool has_sufficient_progbuf(struct target *target, unsigned size)
 	return info->progbufsize + r->impebreak >= size;
 }
 
-/**
- * Immediately write the new value to the requested register. This mechanism
- * bypasses any caches.
+/** This function recives a program that stores a register at a memory address
+ * stored at S0, allocates the memory needed, executes it and reads the value
+ * of the register.
  */
```

Comment:
Detail: Please start the comment text on the second line, not directly after `/**`.

Reasons:
- Consistency with other comments
- Consistency with the recommended doxygen style (https://www.doxygen.nl/manual/docblocks.html)

(There are multiple occurrences in this merge request - I won't comment on each.)

```suggestion
/** 
 * This function recives a program that stores a register at a memory address
 * stored at S0, allocates the memory needed, executes it and reads the value
 * of the register.
 */
```

Reply:
Addressed.